### PR TITLE
Enable parallel write

### PR DESCRIPTION
**Pull request by marscher**
in #9 parallel reading was enabled. Now lets do the same for writing. I've tested this with my (large) project locally. The output looks sane and the total build time was cut in half.

### CODE REVIEW

**Review by gaborbernat — CHANGES_REQUESTED** [review]
Can we add a changelog entry for this?

**Comment by marscher** [comment]
From my observation it does not, otherwise I could not explain why the build time decreases to 50%. This was taken with Sphinx 5.3

**Comment by hoodmane** [comment]
Well the change is either harmless or useful so it isn't a big deal to merge it. But the code for v5.3.0 also seems to show that this defaults True:
```
         self.parallel_write_safe = kwargs.pop('parallel_write_safe', True)
```
https://github.com/sphinx-doc/sphinx/blob/v5.3.0/sphinx/extension.py#L33

used here:
https://github.com/sphinx-doc/sphinx/blob/v5.3.0/sphinx/application.py?plain=1#L1278

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## unreleased
```
Comment (gaborbernat) [inline CHANGELOG.md:3]: 1.22

Comment (marscher) [reply CHANGELOG.md]: there you go.

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**